ex18_praticepython.py

```diff
@@ -20,12 +20,19 @@ def compare(lottery,guess):
     score = 0
 
     while playing:
-        guess = list(input("Give me your best guess! The digits must be all different. If the matching digits are in their right positions, they are bulls, if in different positions, they are cows."))
+        guess = list(input("Give me your best guess! The digits must be all different. If the matching digits are in their right positions, they are bulls, if in different positions, they are cows.\n"))
         guess = [int(i) for i in guess]
+        guess=set(guess)
+        guess=list(guess)
+        if len(guess)!=4:
+            guess = list(input("You didnt put unique values. Try one more time and remember the digits must be all different. If the matching digits are in their right positions, they are bulls, if in different positions, they are cows.\n"))
+        else:
+            continue

```

Comment:
1) Obecnie nie działa całkowicie. Dodałem sobie print(lottery):
```
$ python3 ex18_praticepython.py 
Secret: [8, 2, 0, 3]
Give me your best guess! The digits must be all different. If the matching digits are in their right positions, they are bulls, if in different positions, they are cows.
8203
Give me your best guess! The digits must be all different. If the matching digits are in their right positions, they are bulls, if in different positions, they are cows.
```
Hint: "Python also includes a data type for sets. A set is an unordered collection with no duplicate elements."

2) Ponieważ nie sprawdzasz ile cyfr ma podana liczba mogę podać: 11223344 i program sprawdzi czy trafiłem bez 
wyświetlania komunikatu o błędzie. Jak rozwiążesz problem wyżej stanie się to istotne.